### DISCUSSION TITLE
Why Shovel instead of Scoop ?

### DISCUSSION THREAD

**Discussion by rkbadatya** [Ash258/scoop#173] [Q&A] [answered]
Hi Contributors and Users, I couldn't find much difference between shovel and scoop from the readme section except the SCOOP_REPO configuration change. I have recently started using Scoop. It could be better if there was more information what is the workflow difference between both and if i choose to use Shovel, the CLI I have to use is `shovel install <app>` or `scoop install <app>` ?

**Comment by Ash258** [comment, marked as answer]
https://github.com/Ash258/Scoop-Core/blob/main/CHANGELOG.md

https://github.com/Ash258/Scoop-Core/releases
